### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by cnasikas**

### CODE REVIEW

File: app/src/views/Assignments/Assignment.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ class Assignment extends React.Component {
 
   submitSolution (e) {
     e.preventDefault()
-    const data = new FormData(e.target)
+    const data = new FormData(e.target) // eslint-disable-line
```
Comment (dionyziz) [inline app/src/views/Assignments/Assignment.js]: What's the problem here?

Comment (cnasikas) [reply app/src/views/Assignments/Assignment.js]: `FormData` is a browser library and the linter does not understand it

Comment (dionyziz) [reply app/src/views/Assignments/Assignment.js]: I think you can introduce it as a global var in eslint.

Comment (cnasikas) [reply app/src/views/Assignments/Assignment.js]: You are right. It will be fixed in a future PR

**Pull request merged**